content/board_recruiting.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Information about applying for the board which we publicize every three years when the board turns over.
+---
+
+# Board Recruiting
+
+Open positions will advertised in the months preceding the society’s Annual General Meeting.
+
+## Board Composition
```

Comment:
also I think there are some typos in here, right?

`Compentencies` => `Competencies`
`excercises` => `exercises`
`Visonary` => `Visionary`

Reply:
omg this is my horrible spelling, please fix if you have time hahaha